### ISSUE TITLE
NameError (uninitialized constant Mobility::Backends::ActiveRecord::Container)

### ISSUE THREAD

**Issue by semirart** [shioyama/mobility#173]
Rails 5.1, Mobility 0.4.2

config/initializers/mobility.rb
```ruby
Mobility.configure do |config|
  config.default_backend = :container
  config.accessor_method = :translates
  config.query_method    = :i18n
  config.default_options[:locale_accessors] = true
end
```

The error happens on the first load of any model with `translates` method. In development I can run rails console, type the model class and I'll see this error, on the second call it will work. In production the application doesn't boot of course.

The current workaround is to prepend the initializer with these lines:
```ruby
require "mobility/backends/active_record/jsonb"

# without this everything seems to be working,
# but Mobility::Backends::ActiveRecord::Container 
# is still unitialized when testing in console
require "mobility/backends/active_record/container" 
```

**Comment by shioyama** [comment]
There was a missing `require`, fixed in #174. I'll check specs pass and merge + cut a new release.

**Comment by shioyama** [comment]
This is fixed in 0.4.3 which I just released: https://rubygems.org/gems/mobility/versions/0.4.3
